code/game/verbs/who.dm

```diff
@@ -61,7 +63,7 @@
 	for (var/line in sortList(Lines))
 		msg += "[line]\n"
 	if(holder)
-		msg += "<b>Living: [living] | Dead/Ghosts: [ghosts] | Living Antags: [livingAntags] | </b>"
+		msg += "<b>Living: [living] | Dead/Ghosts: [ghosts] | in Lobby: [lobby] | Living Antags: <font color='red'>[livingAntags]</font> | </b>"
```

Comment:
do we have a font that just changes the color? if you answer "warning" or "rose" or "danger" you lose.